input/pagecontent/StructureDefinition-au-erequesting-servicerequest-imag-intro.md

```diff
@@ -2,7 +2,7 @@
 
 ### Profile specific implementation guidance
 - `ServiceRequest.code`:
-  - **SHOULD** be selected from the preferred RANZCR Radiology Referral value set
-  - if a system is unable to provide a code from the preferred RANZCR Radiology Referral value set , it is preferred to select a suitable code from the SNOMED CT <a href="https://healthterminologies.gov.au/fhir/ValueSet/imaging-procedure-1">Imaging Procedure</a> value set over only supplying local codes
+  - **SHOULD** be selected from the preferred <a href="ValueSet-ranzcr-radiology-procedures.html">RANZCR Radiology Procedures</a> value set
+  - if a system is unable to provide a code from the preferred <a href="ValueSet-ranzcr-radiology-procedures.html">RANZCR Radiology Procedures</a> value set , it is preferred to select a suitable code from the SNOMED CT <a href="https://healthterminologies.gov.au/fhir/ValueSet/imaging-procedure-1">Imaging Procedure</a> value set over only supplying local codes
```

Comment:
Please remove the extra space before the comma.

Reply:
Comma removed.